setup.py

```diff
@@ -1,26 +1,3 @@
 from setuptools import setup
 
-setup(
-    name="conda-smithy",
```

Comment:
There is a long standing bug in GitHub that only reads the name metadata of Python packages from setup.py. Not that we need it but removing this line from here will mess up the info in https://github.com/conda-forge/conda-smithy/network/dependents.

PS: really not a big deal for conda-smithy but something to consider for other packages.

Reply:
Ah ok. I propose we let this bug be a bug and remove this.